includes/class-freemius.php

```diff
@@ -918,9 +958,49 @@ private function _find_caller_plugin_file( $is_init = false ) {
 				'path' => $id_slug_type_path_map[ $this->_module_id ]['path'],
 			);
 
-			return $id_slug_type_path_map[ $this->_module_id ]['path'];
+			return $this->get_absolute_path( $id_slug_type_path_map[ $this->_module_id ]['path'] );
 		}
 
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 1.2.2.11
+         *
+         * @param string $path
+         *
+         * @return string
+         */
+		private function get_relative_path( $path ) {

```

Comment:
@fajardoleo this method is confusing. It gets a `$path` argument but it doesn't really work for any path beside the path of the current context module.